superset/reports/commands/alert.py

```diff
@@ -46,6 +47,10 @@
 # to avoid heavy loads done by a user mistake
 OPERATOR_FUNCTIONS = {">=": ge, ">": gt, "<=": le, "<": lt, "==": eq, "!=": ne}
 
+# Max tries to run queries to prevent false errors caused by transient errors
+# being returned to users
+EXECUTE_QUERY_MAX_TRIES = 2
```

Comment:
keep it at 1 to keep the default behavior
also let's move it to the config to let other to decide what retry policy they are looking for